MekHQ/src/mekhq/gui/dialog/PartsReportDialog.java

```diff
@@ -127,6 +130,43 @@ public void actionPerformed(ActionEvent e) {
                 refreshOverviewSpecificPart(row, piu, partToBuy);
             }
         };
+
+        Action sell = new AbstractAction() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                int row = Integer.parseInt(e.getActionCommand());
+                PartInUse piu = overviewPartsModel.getPartInUse(row);
```

Comment:
I'd prefer if this were `partsInUse`, this is just personal preference, but I've had to dig through enough old code to know that no matter how obvious an abbreviated variable name is when you write in; in ten years time nobody is going to know at a glance what it is.

Reply:
No worries, done in cdc6ddb :)